corehq/apps/app_manager/templates/app_manager/apps_base.html

```diff
@@ -39,7 +39,7 @@
     <script src="{% new_static 'bootstrap3-typeahead/bootstrap3-typeahead.min.js' %}"></script>
     <script src="{% new_static 'style/js/bootstrap-multi-typeahead.js' %}"></script>
 
-    <script src="{% new_static 'hqwebapp/js/ui-element.js' %}"></script>
+    <script src="{% new_static 'style/js/bootstrap2/ui-element.js' %}"></script>
```

Comment:
it was still using the old file which got moved to `style/js/bootstrap2/ui-element.js`, wanted to keep the switchover separate